docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -1,31 +1,146 @@
+    | Resource       | Display Acquired Images | Disabled                                                                                                                                                            |
+    | Resource       | Log I2C to Disk         | Enabled                                                                                                                                                             |
+	| Serial Channel | Configuration Script    | [Refer to Configuration Script in the PXIe-148X Acquisition GSE Help](../../reference/gettingstartedexample/gse-acq-help.md#table-of-pxie-148x-acquisition-scripts) |
+    
+    | Acquisition    | Continuous Acquisition  | Disabled                                                                                                                                                            |
+    | Board          | Power Over Coax Source  | Internal                                                                                                                                                            |
+4.  Select the **I2C** tab and make the following modifications.
+
+    > The I2C tab only has one control, the I2C **timestamp filter**. This filter contains an array of timestamp IDs. **User24** represents the I2C traffic on serial channel 0 (SI0), **User25** represents the I2C traffic on serial channel 1 (SI1), and so on.
+    - Set the **timestamp filter** array to contain only the **User24** timestamp ID. This will let you see the I2C traffic on SI0. If you run the VI with a configuration script selected, you will see configuration traffic in the I2C Data Output tab after the VI has stopped.
+
+    ![Configuration Settings I2C Tab](../../images/PXIe-148X-Acq-I2CTab-FiniteAcq.png)
+
+5.  Run the Acquisition VI. 
+
+6.  Run the Generation VI.
+
+7.  Select the **I2C Timestamps** tab to view I2C timestamp data.
+    - View the displayed I2C timestamp data in the **I2C Timestamps** table. The **I2C Timestamps** table displays I2C timestamp information for all I2C traffic. I2C timestamps begin logging immediately after the FPGA bitfile is downloaded and include timestamp data prior to the start of the LLP packet data acquisition (i.e. I2C traffic from the configuration script).
+
+        > Note: To display I2C timestamp data, **Log I2C to Disk** must be enabled and desired timestamp IDs must be added to the **timestamp filter** array. The I2C timestamp data displayed is read from the User_Timestamps.tdms file and filtered to display only timestamp IDs included in the timestamp filer array. The **I2C Timestamps** display is updated after the acquisitions completes.
+
+    ![I2C Timestamps Data](../../images/PXIe-148X-Acq-I2CTimestamps-FiniteAcq.png)
 
 ## Generating and Filtering LLP Packets
+(There is no filtering on the generation side, and the acq tutorial already covers filtering)
```

Comment:
We should delete this section but I have left it here because it was part of our original outline. I just wanted people to be aware of it.